ArticleTemplates/assets/js/modules/quiz.js

```diff
@@ -132,20 +131,39 @@ define([
                 }
             },
 
-            getCorrectAnswers: function () {
+            getQuestionsAndAnswers: function () {
                 var i,
-                    answers = [],
-                    correctAnswers = document.querySelector('.quiz__correct-answers').innerHTML.split(','),
-                    correctAnswerArray,
-                    correctAnswerObj;
-                
-                for (i = 0; i < correctAnswers.length; i++) {
-                    correctAnswerObj = {};
-                    correctAnswerArray = correctAnswers[i].split(':')[1].split('-');
-                    correctAnswerObj.code = correctAnswerArray[0].trim().toUpperCase();
-                    correctAnswerObj.explanation = correctAnswerArray[1] || '';
-                    answers.push(correctAnswerObj);
-                } 
+                    key,
+                    answerMatch,
+                    question,
+                    answer,
+                    answers = {},
+                    correctAnswerWordList = document.querySelector('.quiz__correct-answers').innerHTML.split(' ');
```

Comment:
I know you've not changed this but this feels like something that should be embedded in the page as JSON rather than doing some icky splitting and regexing. I'm sure you've thought about this already